api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -0,0 +1,116 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    body {
+      background-color: #F1F1F1;
+      padding-left: 1rem;
+      padding-top: 1rem;
+      width: 600px;
+    }
+    * {
+      font-family: "Gotham SSm A", "Gotham SSm B", sans-serif;
+    }
+    .welcome-message {
+      margin-top: 2rem;
+      margin-bottom: 1rem;
+      color: #302C71;
+      font-size: 30px;
+      font-weight: 300;
+      line-height: 36px;
+    }
+
+    .text {
+      color: rgba(0,0,0,0.87);
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 22px;
+      margin-bottom: 1rem;
+    }
+
+    .text.user-field {
+      width: 8rem;
+      display: inline-block;
+      margin-bottom: 0;
+    }
+    .user-info {
+      font-weight: bold;
+      display: inline-block;
+    }
+    .card {
+      height: 70px;
+      width: 560px;
+      border-radius: 4px;
+      background-color: #FFFFFF;
+      box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16);
+      display: flex;
+      justify-content: flex-start;
+      align-items: center;
+      margin-bottom: 1rem;
+      cursor: pointer;
+      text-decoration: none;
+    }
+
+    div.card {
+      cursor: default;
+    }
+
+    .bullet {
+      height: 31px;
+      margin-left: 1rem;
+      margin-right: 1rem;
+      width: 31px;
+      border-radius: 15px;
+      background-color: #302C71;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      color: #FFFFFF;
+    }
+
+    .card-text {
+      color: #302C71;
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 19px;
+      width: 460px;
+    }
+    .link {
+      display: inline;
+      color: #2691D0;
+    }
+    .button {
+      height: 52px;
+      width: 178px;
+      border-radius: 5px;
+      background-color: #302C71;
+      font-size: 16px;
+      font-weight: 500;
+      line-height: 19px;
+      color: #FFFFFF;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      text-transform: uppercase;
+      text-decoration: none;
+    }
+  </style>
+</head>
+<body>
+  <img src="assets/logo-registration-non-signed-in.svg">
+  <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
+  <div><div class="text user-field">Your User Name: </div><div class="user-info">{{USERNAME-INPUT}}</div></div>
+  <div><div class="text user-field">Your Password: </div><div class="user-info">{{PASSWORD-INPUT}}</div></div>
+  <br>
+  <div class="text">Quick guide on how to get started:</div>
+  <a class="card" href="https://myaccount.google.com/signinoptions/two-step-verification">
```

Comment:
I foresee many people just filling out their normal gmail account here and definitely not remembering the exact research domain name for the account they just created. At this point in the flow I don't think the research account would even show up in the list of Google accounts. I think we still do need a more directed link here somehow.

Reply:
We're actually going to be handing them a full gmail username for the username above, so I think that it might be okay? I would like to autopopulate if we can, but I don't know how to do that... (Especially when routing to a google security page... It handles its own sign in, so we couldn't even restrict by domain...)

Reply:
Yeah fine to iterate on this. I don't think we need user feedback to know that this link is not ideal (though user feedback may tell us if it is unacceptably bad). My personal expectation is that people will gloss over a lot of this content and mindlessly login to their usual accounts, then get stuck.

Reply:
Yeah. If it is unacceptably bad, then another solution would be good. In the meanwhile I am going to say we put this in, and then keep looking to see if anyone knows how to restrict domain on the security page login. I couldn't figure out how, but if its possible, that would probably help considerably. I can reach out to some of the people I know in google, but may be good for you to ask around as well if you can.